components/cycles/CycleOrder.jsx

```diff
@@ -0,0 +1,128 @@
+import { View } from 'react-native';
+import { FlatList } from 'react-native-gesture-handler';
+import { IconButton } from 'react-native-paper';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+import * as React from 'react';
+
+const CycleTitle = styled.Text`
+   color: #FFFFFF;
+   font-size: 24px;
+   font-family: 'Montserrat_500Medium';
+   textAlign: left;
+   paddingLeft: 15px;
+   paddingTop: 5px;
+`;
+
+const SubTitle = styled(CycleTitle)`
+    font-size: 16px;
+    paddingBottom: 5px;
+    font-family: 'Montserrat_600SemiBold';
+`;
+
+const WorkoutViewText = styled.Text`
+    color: #FFFFFF;
+    font-family: 'Montserrat_500Medium';
+    font-size: 12px;
+    paddingTop: 2px;
+    paddingBottom: 4px;
+    width: 33%;
+    textAlign: center;
+`;
+
+const RowHeader = styled.View`
+    flexDirection: row;
+    alignContent: center;
+    font-family: 'Montserrat_500Medium';
+    justifyContent: space-evenly;
+    width:100%;
+`;
+
+const RowContent = styled.View`
+    flexDirection: row;
+    alignContent: center;
+    justifyContent: space-evenly;
+`;
+
+const CycleOrderCard = (props) => {
+  const [showDetailed, setshowDetailed] = React.useState(false);
+  const [icon, setIcon] = React.useState('chevron-down');
+
+  const {
+    name, muscleGroups, color, exercises,
+  } = props;
+  const FullBody = styled.View`
+        width: 90%;
+        margin: auto;
+        background-color: ${color};
+        borderRadius: 10px;
+        box-shadow: 1px 5px 2px gray;
+        marginBottom: 10px;
+        marginTop: 10px;
+        padding: 10px;
+    `;
+
+  const ExercisesView = (items) => {
+    const workoutItems = items.items;
```

Comment:
The function that represents a component should always take in "props", also can we rename items and workoutItems to something more descriptive, say exercises.
```suggestion
  const ExercisesView = (props) => {
    const { exercises } = props;
```